pages/api/jtline/stations/index.ts

```diff
@@ -22,16 +22,15 @@ const jtlineStopsQuery = `with jtline_stops as
           and now() between start_date and end_date
           and ST_Distance(
                   the_geom::geography,
-                  ST_SetSRID(ST_Point(24.9533522, 60.1672325), 4326)::geography
-                ) < 10000)
+                  ST_SetSRID(ST_Point($1, $2), 4326)::geography
```

Comment:
Is there some magic here that makes this invulnerable to SQLI?

Reply:
It's a parameterised query as described here: https://node-postgres.com/features/queries